platform/platform-app/pom.xml

```diff
@@ -323,13 +323,13 @@
         DO NOT EDIT UNLESS UPDATING CXF-->
         <!-- Copied from apache-cxf-3.1.4-src/parent/pom.xml -->
         <cxf.abdera.version>1.1.3</cxf.abdera.version>
-        <cxf.activemq.version>5.12.0</cxf.activemq.version>
+        <cxf.activemq.version>5.12.2</cxf.activemq.version>
```

Comment:
This property isn't used in `platform/platform-app/src/main/resources/features.xml`, so it shouldn't need to be updated here.